nautobot/dcim/tests/test_models.py

```diff
@@ -688,6 +689,11 @@ class InterfaceTemplateTestCase(ModularDeviceComponentTemplateTestCaseMixin, Tes
     modular_component_create_data = {"type": InterfaceTypeChoices.TYPE_1GE_FIXED}
     model = InterfaceTemplate
 
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.device_type = DeviceType.objects.filter(interface_templates__isnull=False).first()
```

Comment:
Not sure I see why the `.filter()` is necessary here, but I don't think it's necessarily hurting anything either.